README.md

```diff
@@ -1,42 +1,74 @@
 # zkLLVM Template
 
-This repository contains the template repository for the project using zkLLVM as a compiler. 
+Tutorial and a template repository for a zk-enabled application project
+based on the [zkLLVM toolchain](https://github.com/nilfoundation/zkllvm).
 
-The repository contains a template code to prove a BLS signature via zkLLVM using [Crypto3 C++ cryptography suite](https://github.com/nilfoundation/crypto3) as an SDK.
+Use it to learn about developing zk-enabled apps with zkLLVM step-by-step.
 
-# Dependencies
+Example code in `./src` proves a BLS signature via zkLLVM using 
+[Crypto3 C++ cryptography suite](https://github.com/nilfoundation/crypto3) as an SDK.
 
-- [Boost](https://www.boost.org/) == 1.76.0
```

Comment:
Boost is temporarily required by the SDK's CMake configs, but is not actually used by zkLLVM pipeline. Thus we still need to have it installed on the machine